pallets/parachain-staking/src/lib.rs

```diff
@@ -1476,6 +1440,20 @@ pub mod pallet {
 			<EnableMarkingOffline<T>>::set(value);
 			Ok(())
 		}
+
+		/// Force join the set of collator candidates.
+		/// It will skip the minimum required bond check.
+		#[pallet::call_index(31)]
+		#[pallet::weight(<T as Config>::WeightInfo::join_candidates(*candidate_count))]
+		pub fn force_join_candidates(
+			origin: OriginFor<T>,
+			acc: T::AccountId,
```

Comment:
`acc` is a shortcut for `accumulator`, it's too confusing, please rename it account

Reply:
done